cluster/manifests/wiz/sensor/secrets.yaml

```diff
@@ -0,0 +1,39 @@
+{{ if eq .Cluster.ConfigItems.wiz_enable_runtime_monitoring_daemonset "true"}}
+---
+# Source: wiz-sensor/templates/apikeysecret.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: wiz-sensor-apikey
+  namespace: wiz
+  labels:
+    
+    helm.sh/chart: wiz-sensor-1.0.4760
+    image/tag: v1
+    app.kubernetes.io/name: wiz-sensor
+    app.kubernetes.io/instance: wiz-sensor
+    app.kubernetes.io/version: "1.0.4708"
+    app.kubernetes.io/managed-by: Helm
+type: Opaque
+data:
+  clientId: "{{ .Cluster.ConfigItems.wiz_api_client_id | base64 }}"
+  clientToken: "{{ .Cluster.ConfigItems.wiz_api_client_token | base64 }}"
+---
+# Source: wiz-sensor/templates/imagepullsecret.yaml
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/dockerconfigjson
+metadata:
+  name: wiz-sensor-imagepullkey
+  labels:
+    
+    helm.sh/chart: wiz-sensor-1.0.4760
+    image/tag: v1
+    app.kubernetes.io/name: wiz-sensor
+    app.kubernetes.io/instance: wiz-sensor
+    app.kubernetes.io/version: "1.0.4708"
+    app.kubernetes.io/managed-by: Helm
+  namespace: wiz
+data:
+  .dockerconfigjson: "{{ .Cluster.ConfigItems.wiz_sensor_dockerconfigjson }}"
```

Comment:
This looks strange 😅  I suggest we do the following:
1. Instead of a Secret, make this manifest a `ConfigMap` that keeps the docker config JSON.
2. We expose only the token itself as a config-item, instead of `wiz_sensor_dockerconfigjson`. (or username and token both can be config-items)
3. We populate the token in the Docker config JSON using that config-item in the ConfigMap instead.